app/views/unattended/provisioning_templates/snippet/ansible_tower_callback_script.erb

```diff
@@ -10,9 +10,17 @@ description: |
 #!/bin/sh
 
 echo "Calling Ansible AWX/Tower provisioning callback..."
+
+# Query the AC version
+if [ $(echo "$(curl -k -s https://<%= host_param('ansible_tower_fqdn') %>/api/v2/ping/ | jq -r .version | cut -d. -f1,2) >= 4.6" | bc) -eq 1 ]; then
```

Comment:
This runs on the machine being deployed, so I don't think we can rely on `jq` being available

Reply:
Probably very fragile, but would it make sense to query the version on the Foreman side and then render the correct template?

Another thought: if you're going to rely on `/usr/bin/python3` anyway, rewrite the whole script to Python instead of bash?

Third option: first post to the new API endpoint and if that fails, post to the old API endpoint.

Reply:
I think I prefer the third option.

Reply:
The new implementation (while untested) fixes my concerns, thanks!